target/linux/mvebu/config-4.9

```diff
@@ -60,6 +60,8 @@ CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_BLK_MQ_PCI=y
 CONFIG_BOUNCE=y
+CONFIG_BTRFS_FS=y
```

Comment:
> I would at least like to avoid requiring use of a serial cable for this process (though at least the pin header is populated in the factory!)

Does the bootloader have an automated tftp recovery mode? If it does you can just send it an initramfs image and then you can ssh in and do a sysupgrade (which will reformat if needed).
If you have a sysupgrade working, anyway.